tests/integration/test_enrollment/test_agentd_enrollment.py

```diff
@@ -0,0 +1,84 @@
+'''
+brief: This module verifies the correct behavior of Wazuh Agentd during the enrollment under different configurations.
+copyright:
+    Copyright (C) 2015-2021, Wazuh Inc.
+    Created by Wazuh, Inc. <info@wazuh.com>.
+    This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+metadata:
+    component:
+        - Agent
+    modules:
+        - Agentd
+    daemons:
+        - Agentd
+    operating_system:
+        - Ubuntu
+        - CentOS
+        - Windows
+    tiers:
+        - 0
+    tags:
+        - Enrollment
+        - Agentd
+'''
+
+import pytest
+import ssl
+import yaml
+from wazuh_testing.tools.monitoring import ManInTheMiddle
```

Comment:
There is no need to import these if we aren't using them

Reply:
Fixed